tests/test_indexer.py

```diff
@@ -445,8 +447,17 @@ def _get_version(self):
 class PostTestHandler(BaseHTTPRequestHandler):
     _response_code = 200
     _payload = None
+    hmac_secret_key = "ribos0me"
 
     def do_POST(self):
+        HTTPSignatureAuth.verify(requests.Request("POST", self.path, self.headers),
+                                 key_resolver=lambda key_id, algorithm: self.hmac_secret_key.encode())
+        try:
+            HTTPSignatureAuth.verify(requests.Request("POST", self.path, self.headers),
+                                     key_resolver=lambda key_id, algorithm: self.hmac_secret_key[::-1].encode())
+            raise Exception("Expected AssertionError")
```

Comment:
should just be `self.fail(..)`

Reply:
This is not a unittest.TestCase subclass.

Reply:
unittest.TestCase is inherited at run time through it's children.

Reply:
I'm very confused about what you mean, @Bento007...

Reply:
My mistake, I thought this was a modification to `TestIndexerBase` which is an abstract class that uses unittest.TestCase asserts.